cmake/modules/FindGnuLd.cmake

```diff
@@ -43,9 +43,11 @@ if(EXISTS "${GNULD_LINKER}")
 endif()
 
 # See if the compiler has a preferred linker
-execute_process(COMMAND ${CMAKE_C_COMPILER} --print-prog-name=ld.bfd
-                OUTPUT_VARIABLE GNULD_LINKER
-                OUTPUT_STRIP_TRAILING_WHITESPACE)
+execute_process(COMMAND
+  ${CMAKE_C_COMPILER} --print-prog-name=ld.bfd
```

Comment:
Why is this line wrapped ?
```
COMMAND ${CMAKE_C_COMPILER} --print-prog-name=ld.bfd
```

on a single line makes perfect sense, because `COMMAND` is the argument name to the function, and the following are values to the `COMMAND` argument.

Reply:
I tried to follow the example and give the first keyword more importance, updating this to the other style.